coordinator/tests/services/validation/bundle-validator.spec.ts

```diff
@@ -352,7 +352,7 @@ describe("verifyDispenseNotificationBundle", () => {
 
   test("returns an error when MedicationDispenses have different performer values per type", () => {
     const medicationDispenseEntry =
-      bundle.entry.filter(entry => entry.resource.resourceType === "MedicationDispense")[0]
+      clone(bundle.entry.filter(entry => entry.resource.resourceType === "MedicationDispense")[0])
```

Comment:
can we use the `getMedicationDispenses` here?

Reply:
Not quite, it's one level above, getting the whole entry rather than the resource itself 👍